dllNetwork/server/ServerI2CInstruction.C

```diff
@@ -472,20 +523,8 @@ ssize_t ServerI2CInstruction::iic_read(Handle * i_handle, ecmdDataBufferBase & o
 
         uint8_t * buffer = new uint8_t[len + 4];
 
-        // adjust timeout value based on data size
-        int timeout = len * 160 * 2 + 1000;
 #ifdef SERVER_PRINT_PERF
-        printf("bytes: %d, timeout: %d\n", len, timeout);
-#endif
-#ifdef TESTING
-        TESTPRINT("rc = adal_iic_config((adal_t *) i_handle, ADAL_CONFIG_WRITE, ADAL_IIC_CFG_TIMEOUT, %x, 0);\n", timeout);
-#else
-        ssize_t rc = adal_iic_config((adal_t *) i_handle, ADAL_CONFIG_WRITE, ADAL_IIC_CFG_TIMEOUT, &timeout, 0);
-        if (rc)
-        {
-            delete [] buffer;
-            return rc;
-        }
+        printf("bytes: %d, msDelay: %d, numRedos: %d\n", len, msDelay, numRedos);
```

Comment:
need to remove this debug code

Reply:
This is only within the SERVER_PRINT_PERF and is expanding what was there previously.  
Still want it removed?

Reply:
sorry missed that, should be fine then.